src/store/account-details/account-details.thunks.ts

```diff
@@ -142,8 +142,8 @@ function filterExitsFromHistoryTransactions(
 // ToDo: Define fromItem type
 function fetchHistoryTransactions(
   accountIndex: HermezAccount["accountIndex"],
-  fromItem: number,
-  exits: Exits
+  exits: Exits,
+  fromItem?: number
```

Comment:
Since you have changed the order of these params, now the function `mapDispatchToProps` on `src/views/account-details/account-details.view.tsx` is wrongly naming the params.